2-mandatory/6-writing-tests-advanced.js

```diff
@@ -52,18 +49,34 @@ function formatCourseworkResult(trainee) {
   Write a test that checks the output of formatCourseworkResult when passed the following trainee:
   {
     name: "Xin",
-    score: 63
+    score: 63 C
   }
 */
-
+test("Trainee name Xin, score C", () => {
```

Comment:
This test looks good, but it's usually a good idea to give the test a name which describes the scenario it is testing. Like in this case, maybe `A trainee with a score of 63 should receive a grade of C` (just an example).